i18n/i18n.jsx

```diff
@@ -217,19 +201,13 @@ export function safariFix(callback) {
 }
 
 export function doAddLocaleData() {
```

Comment:
Maybe split this out via dynamic import is much better.

Reply:
Nah, these inline requires already split chunks and load it conditionally. Since localization data is one of the critical resources, no need to be asynchronous.